code/backend/pages/01_Ingest_Data.py

```diff
@@ -14,6 +14,7 @@
     UserDelegationKey,
 )
 import urllib.parse
+import urllib.request
```

Comment:
Is this needed? I can't see any additions using it.

Reply:
Surprised a precommit didn't catch this